mod_ci/controllers.py

```diff
@@ -52,13 +64,15 @@ class Status:
     FAILURE = "failure"
 
 
-@mod_ci.before_app_request
-def before_app_request():
+@mod_ci.before_app_request  # type: ignore
+def before_app_request() -> None:
     """Organize menu content such as Platform management before request."""
     config_entries = get_menu_entries(
         g.user, 'Platform mgmt', 'cog', [], '', [
-            {'title': 'Maintenance', 'icon': 'wrench', 'route': 'ci.show_maintenance', 'access': [Role.admin]},
-            {'title': 'Blocked Users', 'icon': 'ban', 'route': 'ci.blocked_users', 'access': [Role.admin]}
+            {'title': 'Maintenance', 'icon': 'wrench',
+             'route': 'ci.show_maintenance', 'access': [Role.admin]},  # type: ignore
```

Comment:
It seems illogical to me to add a ignore here?

Reply:
No, it is not illogical. What happens is that the dictionary has most elements in `'str', 'str'` format but last one is `'str', 'Enum'` and hence it gives error, which is not required.

I've enabled a check for notifying of ignore statements which are unnecessary :D

Reply:
Please resolve if you are convinced with the response.